<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QLineEdit" name="leAddres">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPort">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>1234</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDisconnect">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCountClients">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRoot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Root</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTemp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>67</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbConnect</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>OnConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDisconnect</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>OnDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddNew</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>OnAddNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnConnect()</slot>
  <slot>OnDisconnect()</slot>
  <slot>OnAddNew()</slot>
  <slot>OnClearModels()</slot>
 </slots>
</ui>
